client/app/config/services.yml

```diff
@@ -260,7 +260,7 @@ services:
         base_uri: "%api_base_url%"
         verify: false
         headers:
-          Content-Type: application/json
+          Content-Type: app/json
```

Comment:
Find and replace issue?

Reply:
Yeah, this isn't ready for review yet. That button shouldn't have been clicked.

Reply:
Mea culpa - Jim and I misunderstood each other